charmcraft/commands/store/client.py

```diff
@@ -24,13 +24,17 @@
 import appdirs
 from macaroonbakery import httpbakery
 
+from charmcraft import __version__
 from charmcraft.cmdbase import CommandError
 
 logger = logging.getLogger('charmcraft.commands.store')
 
 # XXX Facundo 2020-06-19: only staging for now; will make it "multi-server" when we have proper
 # functionality in Store's production (related: issue #51)
-BASE_URL = 'https://api.staging.snapcraft.io/publisher/api'
+API_BASE_URL = 'https://api.staging.snapcraft.io/publisher/api'
+
+# XXX Facundo 2020-06-29: we need to include here at least the platform, maybe architecture, etc
+USER_AGENT = "charmcraft/{}".format(__version__)
```

Comment:
I very much appreciate this
Also relevant for future KPIs

Reply:
BTW, I opened #74 for this.